lrc/Pos/Lrc/DB/Seed.hs

```diff
@@ -0,0 +1,46 @@
+-- | We want store each epoch's seed in the database. This module does that.
+--
+-- It's done so that we'd be able to reuse previous epoch's seed in case we
+-- can't generate a seed for current epoch (see [CSL-50]).
+module Pos.Lrc.DB.Seed
+       ( -- * Getters
+         getSeed
+
+         -- * Operations
+       , putSeed
+
+         -- * Initialization
+       , prepareLrcSeed
+       ) where
+
+import           Universum
+
+import           Pos.Binary.Class  (encodeStrict)
+import           Pos.Core.Types    (EpochIndex (..), SharedSeed)
+import           Pos.DB.Class      (MonadDB)
+import           Pos.DB.Error      (DBError (DBMalformed))
+import           Pos.Lrc.DB.Common (getBi, putBi)
+import           Pos.Lrc.Genesis   (genesisSeed)
+import           Pos.Util.Util     (maybeThrow)
+
+getSeed :: MonadDB m => EpochIndex -> m SharedSeed
+getSeed epoch =
+    maybeThrow (DBMalformed "Seeds part of LRC DB is not initialized") =<<
+    getBi (seedKey epoch)
```

Comment:
I disagree. If I use `getSeed 100500100500` it doesn't mean that DB is malformed. In initialization we put it only for 0-th epoch. Why not return `Maybe` as it is done everywhere else?